packages/kerosene-test/src/test-id/createTestIds.ts

```diff
@@ -0,0 +1,12 @@
+export function createTestIds<N extends string>(
```

Comment:
Similar to `react-hot-loader`, I think this would be included in `dependencies`, not `devDependencies`. Do you think this function would be better as part of `@kablamo/kerosene-ui` so we're not pulling in testing dependencies like sinon in transitive `dependencies` of consuming packages?

Reply:
yeah I wasn't sure where to place this either. My initial thought was to put it in `@kablamo/kerosene-ui`. I'll move it back there.